### PR TITLE
Transport Permits for SBC Users

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#20127

*Description of changes:*
- Transport Permits for SBC Users
- Unit tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/composables/mhrInformation/useTransportPermits.ts [hunk 1]
```diff
@@ -109,14 +109,14 @@ export const useTransportPermits = () => {
...
+  const isTransportPermitDisabled = computed((): boolean =>
+    // QS or SBC role check
+    (isRoleQualifiedSupplier.value || isRoleStaffSbc.value) &&
     // PPR Liens check
-    [APIRegistrationTypes.LAND_TAX_LIEN,
+    ([APIRegistrationTypes.LAND_TAX_LIEN,
     APIRegistrationTypes.MAINTENANCE_LIEN,
     APIRegistrationTypes.MANUFACTURED_HOME_NOTICE]
-      .includes(getLienRegistrationType.value as APIRegistrationTypes) &&
+      .includes(getLienRegistrationType.value as APIRegistrationTypes) ||
```
Comment (dimak1) [inline ppr-ui/src/composables/mhrInformation/useTransportPermits.ts:119]: Liz confirmed that either condition (Lien check or Unit Note) would block the Transport Permit.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1781-42g99fcg.web.app

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1781-42g99fcg.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1781-42g99fcg.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1781-42g99fcg.web.app

**Pull request merged**